internal/gitprovider/result.go

```diff
@@ -43,6 +44,10 @@ func (r *Result) ToMap() map[string]any {
 		m["title"] = r.Title
 	}
 
+	if r.Slug != "" {
+		m["slug"] = r.Slug
+	}
```

Comment:
If we are adding slug, maybe it should be populated for GitLab MR also?

Reply:
If we are adding `slug` as an exported input, I think we should do it consistently for all providers that expose such a prop. This could be done in a followup PR, before we release Flux Operator next.